README.md

```diff
@@ -35,6 +35,7 @@ jobs:
 | `cache-group`      | The group of the cache, defaults to a unique identifier for the workflow job. If you want two jobs to share the same cache, give them the same group name.                                                                                                    | `string`  | `${{ hashFiles(env.workflow_path)-${{ github.job }}-${{ strategy.job-index }}` |
 | `cargo-home`       | The location of the Cargo cache files. If you specify the `CARGO_HOME` env variable for your commands, you need to set it here too. This must NOT end with the trailing slash of the directory.                                                               | `string`  | `~/.cargo`                                                                     |
 | `cargo-target-dir` | Location of where to place all generated artifacts, relative to the current working directory. If you specify the `CARGO_TARGET_DIR` env variable for your commands, you need to set it here too. This must NOT end with the trailing slash of the directory. | `string`  | `target`                                                                       |
+|`manifest-path`|The path to `Cargo.toml`. This is used to determine where `Cargo.lock` is, which is used in the cache key.|`string`|`Cargo.toml`|
```

Comment:
The table format scales rather poorly, I'll create a follow up issue to change that